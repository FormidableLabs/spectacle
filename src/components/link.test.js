import React from 'react';
import { mount } from 'enzyme';
import { mountToJson } from 'enzyme-to-json';
import Link from './link';

describe('<Link />', () => {
  test('should render correctly', () => {
    const context = { styles: { components: { link: { color: '#345' } } } };
    const wrapper = mount((
      <Link
        href="https://www.formidable.com"
        target="_blank"
      >
        Formidable Labs
      </Link>
    ), { context });
    expect(mountToJson(wrapper)).toMatchSnapshot();
  });
});
