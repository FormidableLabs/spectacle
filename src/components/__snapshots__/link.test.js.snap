// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> should render correctly 1`] = `
<Link
  href="https://www.formidable.com"
  target="_blank"
>
  <Styled(a)
    className={undefined}
    href="https://www.formidable.com"
    styles={
      Array [
        Object {
          "color": "#345",
        },
        Object {},
        Object {},
        undefined,
      ]
    }
    target="_blank"
  >
    <a
      className="css-qjaoxy"
      href="https://www.formidable.com"
      target="_blank"
    >
      Formidable Labs
    </a>
  </Styled(a)>
</Link>
`;
